src/components/Home/NoticeFunSystem/NoticeFunSystem.tsx

```diff
@@ -0,0 +1,64 @@
+import { useState } from 'react';
+import * as styles from './NoticeFunSystem.style';
+
+interface SummaryText{
+  textContent : string;
+  date : string;
+}
+
+export interface NoticeFunSystemProps{
+  title : string;
+  summaries : SummaryText[];
+}
+
+
+const NoticeFunSystem:React.FC<NoticeFunSystemProps> = ({title, summaries}) => {
+
+  return (
+    <div>
+      <styles.SummaryBox>
+        <styles.TopBox>
+          <styles.Title>
+            {title}
+          </styles.Title>
+          <styles.MoreButton>
+            더보기
+            <img className= "arrow" src = "/assets/icon/Arrow/RightSmallArrow.svg"/>
```

Comment:
이미지는 Nextjs의 이미지를 사용해주세요!
```suggestion
            <Image className= "arrow" src = "/assets/icon/Arrow/RightSmallArrow.svg"/>
```